core/domain/user_services.py

```diff
@@ -341,7 +347,8 @@ def get_users_settings(user_ids):
                 feconf.SYSTEM_COMMITTER_ID,
                 email=feconf.SYSTEM_EMAIL_ADDRESS,
                 username='admin',
-                last_agreed_to_terms=datetime.datetime.utcnow()
+                last_agreed_to_terms=datetime.datetime.utcnow(),
+                role='SUPER_ADMIN'
```

Comment:
Refer to the named constant. Don't use raw strings; they are susceptible to typos.